src/t8_schemes/t8_default/t8_default_prism/t8_default_prism_cxx.cxx

```diff
@@ -476,5 +476,16 @@ t8_default_scheme_prism_c::~t8_default_scheme_prism_c ()
    * However we need to provide an implementation of the destructor
    * and hence this empty function. */
 }
+void
+t8_default_scheme_prism_c::t8_element_root (t8_element_t *elem) const
+{
+  t8_dprism_t *prism = (t8_dprism_t *) elem;
```

Comment:
I think it is better to call the root functions of the tri and the line here.

Reply:
I didn't want to touch 16 additional files, therefore I didn't put the implementation in the `*_bits.c` file. Thus, I cannot call the tri and line functionality here. The same applies for your comment regarding `T8_DTRI_TO_DTET` as I do not have the base functionality which could be compiled once with and once without `T8_DTRI_TO_DTET`. 

I will do it if that is our policy, but I am not completely sure of the benefits to have everything in the `*_bits.c`

Reply:
Ah yes, you are right, I missed that. That is ok for me.